src/Controls/src/Core/RadioButton.cs

```diff
@@ -31,8 +31,6 @@ public partial class RadioButton : TemplatedView, IElementConfiguration<RadioBut
 		TapGestureRecognizer _tapGestureRecognizer;
 		View _templateRoot;
 
-		static readonly Brush RadioButtonCheckMarkThemeColor = ResolveThemeColor("RadioButtonCheckMarkThemeColor");
-		static readonly Brush RadioButtonThemeColor = ResolveThemeColor("RadioButtonThemeColor");
```

Comment:
The RadioButton.Tizen.cs is failing because is using this.

`/Users/builder/azdo/_work/2/s/src/Controls/src/Core/HandlerImpl/RadioButton/RadioButton.Tizen.cs(112,123): error CS0103: The name 'RadioButtonThemeColor' does not exist in the current context [/Users/builder/azdo/_work/2/s/src/Controls/src/Core/Controls.Core.csproj::TargetFramework=net7.0-tizen]`